src/Pages/SignUp.jsx

```
@@ -1,15 +1,270 @@
+                      <>
+                        {faculties.map((faculty) => {
+                          return (
+                            <option
+                              value={faculty._id}
+                              key={faculty.facultyName}
+                            >
+                              {faculty.facultyName}
+                            </option>
+                          );
+                        })}
+                      </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)